pkg/minikube/notify/notify.go

```diff
@@ -82,41 +81,17 @@ func shouldCheckURLVersion(filePath string) bool {
 	return true
 }
 
-type Release struct {
-	Name      string
-	Checksums map[string]string
-}
-
-type Releases []Release
-
-func getJson(url string, target *Releases) error {
+func getLatestVersionFromURL(url string) (semver.Version, error) {
 	r, err := http.Get(url)
 	if err != nil {
-		return errors.Wrap(err, "Error getting minikube version url via http")
+		return semver.Version{}, err
```

Comment:
Should this remain errors.Wrap?  This is actually an issue all over in that we have some errors which are wrapped to have stack tracing and some which are not.